x-pack/platform/plugins/shared/index_management/public/application/sections/home/index_list/index_list.tsx

```diff
@@ -9,24 +9,34 @@ import React, { useEffect } from 'react';
 import { RouteComponentProps } from 'react-router-dom';
 
 import { ScopedHistory } from '@kbn/core/public';
+import useObservable from 'react-use/lib/useObservable';
 import { APP_WRAPPER_CLASS, useExecutionContext } from '../../../../shared_imports';
 import { breadcrumbService, IndexManagementBreadcrumb } from '../../../services/breadcrumbs';
 import { useAppContext } from '../../../app_context';
 import { IndexTable } from './index_table';
 
 export const IndexList: React.FunctionComponent<RouteComponentProps> = ({ history }) => {
   const {
-    core: { executionContext },
+    core: { executionContext, chrome },
+    plugins: { cloud },
   } = useAppContext();
 
   useExecutionContext(executionContext, {
     type: 'application',
     page: 'indexManagementIndicesTab',
   });
-
+  const activeSolutionId = useObservable(chrome.getActiveSolutionNavId$());
   useEffect(() => {
-    breadcrumbService.setBreadcrumbs(IndexManagementBreadcrumb.indices);
-  }, []);
+    if (cloud?.isServerlessEnabled) {
+      breadcrumbService.setBreadcrumbs(IndexManagementBreadcrumb.indices);
+    } else {
+      if (activeSolutionId === 'es') {
+        breadcrumbService.setBreadcrumbs(IndexManagementBreadcrumb.indicesList);
+      } else {
+        breadcrumbService.setBreadcrumbs(IndexManagementBreadcrumb.indices);
+      }
+    }
+  }, [activeSolutionId, cloud]);
```

Comment:
```suggestion
    if (!cloud?.isServerlessEnabled && activeSolutionId === 'es') {
        breadcrumbService.setBreadcrumbs(IndexManagementBreadcrumb.indicesList);
    } else {
        breadcrumbService.setBreadcrumbs(IndexManagementBreadcrumb.indices);
    }
  }, [activeSolutionId, cloud]);
```

Reply:
Good catch. Missed to clean this one

Reply:
updated in 0972fc71a00bc377e941872cca8e4a5eb99b6b98